client/my-sites/customer-home/cards/actions/quick-links-for-devs/index.jsx

```diff
@@ -0,0 +1,131 @@
+import { useTranslate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { useDebouncedCallback } from 'use-debounce';
+import FoldableCard from 'calypso/components/foldable-card';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getPreference } from 'calypso/state/preferences/selectors';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import { getSelectedEditor } from 'calypso/state/selectors/get-selected-editor';
+import isSiteAtomic from 'calypso/state/selectors/is-site-wpcom-atomic';
+import {
+	getSiteFrontPage,
+	getCustomizerUrl,
+	getSiteOption,
+	isNewSite,
+} from 'calypso/state/sites/selectors';
+import getSiteAdminUrl from 'calypso/state/sites/selectors/get-site-admin-url';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import ActionBox from '../quick-links/action-box';
+import '../quick-links/style.scss';
+
+export const QuickLinks = ( {
+	isExpanded,
+	updateHomeQuickLinksToggleStatus,
+	siteSlug,
+	siteAdminUrl,
+	canManageSite,
+	isAtomic,
+} ) => {
+	const translate = useTranslate();
+	const [
+		debouncedUpdateHomeQuickLinksToggleStatus,
+		,
+		flushDebouncedUpdateHomeQuickLinksToggleStatus,
+	] = useDebouncedCallback( updateHomeQuickLinksToggleStatus, 1000 );
+
+	const quickLinks = (
+		<div className="quick-links-for-devs__boxes quick-links__boxes">
+			<ActionBox
+				href={ `/hosting-config/${ siteSlug }#sftp-credentials` }
+				hideLinkIndicator
+				label={ translate( 'Set up SSH' ) }
+				materialIcon="key"
+			/>
+			<ActionBox
+				href={ `/hosting-config/${ siteSlug }#staging-site` }
+				hideLinkIndicator
+				label={ translate( 'Create staging site' ) }
+				gridicon="science"
+			/>
```

Comment:
Should these actions also only be rendered by the `isAtomic` prop, like the Copy site action is?

Alternatively we could change the eligibility logic on the backend to only return this view when the site is atomic.

Reply:
I was thinking about that then I thought maybe show them still when dev creates a free site and let them link to the hosting page to show the upsell banner.

Reply:
That seems fine too. It doesn't really feel like a "quick link" if it takes users to an upsell, but it's consistent with all the other links where we send users to hosting config.